packages/mobile/src/screens/sign-on-screen/components/AccountHeader.tsx

```diff
@@ -161,7 +161,7 @@ export const ProfilePicture = (props: ProfilePictureProps) => {
       variant='strong'
       source={profilePicture ?? { uri: undefined }}
     >
-      {onSelectProfilePicture ? (
+      {onSelectProfilePicture && !profilePicture ? (
```

Comment:
nice, just double checking you can still click the profile image to change image? icon button here seems to be the thing with the touch target..

Reply:
ooh super good catch, fixed it!
Side note, I just spoke with Sammie about making our cover photo press box a bit bigger so I'll follow up with that later